sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample_AnalyzeOperation.cs

```diff
@@ -91,12 +91,12 @@ public void AnalyzeOperation()
                 int docNumber = 1;
                 foreach (RecognizeEntitiesActionResult entitiesActionResults in entitiesActionsResults)
                 {
-                    foreach (RecognizeEntitiesResult result in entitiesActionResults.Result)
+                    foreach (RecognizeEntitiesResult documentsResults in entitiesActionResults.DocumentResults)
```

Comment:
Since we're iterating over a list of `DocumentResults`, would it be better if we called the local variable `documentResult` (no plural)?